.github/workflows/linting.yml

```diff
@@ -8,6 +8,7 @@ on:
       - "**/pyproject.toml"
       - "!haystack/preview/**/*.py"
       - "!test/preview/**/*.py"
+      - "!e2e/preview/**/*.py"
```

Comment:
You need to update `linting-skipper.yml` too.

Reply:
done!